data/ecosystem/ecosystem-packages.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "packagistUrl": "https://packagist.org/packages/akrabat/ip-address-middleware",
+    "githubUrl":  "https://github.com/akrabat/ip-address-middleware",
+    "categories": ["ip", "address", "middleware"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/netglue/laminas-messenger",
+    "githubUrl":  "https://github.com/netglue/laminas-messenger",
+    "categories": ["laminas", "messenger"],
```

Comment:
"laminas" does not make sense here, because we need to differentiate between integration in Mezzio-based and/or laminas-mvc-based applications. This means that the package can be used as a:

- module in a laminas-mvc-based application
- and/or via a config provider in a Mezzio-based application

Maybe the categories are not suitable for this and a separate entry or entries are required.

Reply:
At the moment the "categories" key is only used for filtering in the interface, so I added a few here and there so they'll be visible in the preview - once a new role is found for them, I can upgrade the functionality to fit the request of course.

Even if the current approach of "user defined keywords" is kept, some other guidelines could be defined for them if wanted.

Reply:
we need to brainstorm a bit about those possible categories.